pkg/core/manifest.go

```diff
@@ -84,6 +91,29 @@ func (m *Manifest) VisitResources(f func(resource string) error) error {
 	return nil
 }
 
+// ResourceAbsolutePath takes a path to a resource and returns an equivalent absolute path.
+// If the input path is a http(s) URL or is an absolute file path, it is returned without modification.
+// If the input path is a file URL, the corresponding absolute file path is returned.
```

Comment:
This comment could be a bit misleading: a `file:` URL is absolute by definition, there is no way to express relative paths with a `file:` protocol (because what comes after is the host, even if implicit and after that an *absolute* path, by definition)

Reply:
I'm not sure what is misleading. Does the comment somehow imply that the URL path could be relative?